packages/editor-ui/src/views/SetupView.vue

```diff
@@ -107,8 +108,11 @@ export default defineComponent({
 						await this.uiStore.submitContactEmail(values.email.toString(), values.agree);
 					} catch {}
 				}
+				await initializeAuthenticatedFeatures();
 
-				if (forceRedirectedHere) {
+				if (this.uiStore.leadEnrichmentTemplates) {
```

Comment:
why are we touching setup view? cloud users should never go through this

Reply:
I wasn't aware of that. Ok, we'll do the redirect on the cloud fronted part